### PR TITLE
add-monthly-april-2020

### PR DESCRIPTION
**Pull request by WT-Liu**
Signed-off-by: Wenting Liu <liuwenting@pingcap.com>

### CODE REVIEW

File: content/blog/monthly-april-2020.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: This Month in TiKV - April 2020
+date: 2020-04-30
+author: TiKV Authors
+---
+
+April 1st marked TiKV’s 4th birthday. We are grateful for the support and contribution of the community while focusing on making better TiKV.
```
Comment (nrc) [inline content/blog/monthly-april-2020.md:7]:
```suggestion
April 1st marked TiKV’s 4th birthday. We are grateful for the support and contribution of the community while focusing on making a better TiKV.
```

File: content/blog/monthly-april-2020.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Let’s take a look at what we have achieved in April!
+
+## News
+
+We're working on another API- VerKV, a multi-versioned KV between RawKV and TxnKV, to help TiKV expand application scenarios. The work plan proposed by [@fredchenbj](https://github.com/fredchenbj) is available [here](https://github.com/tikv/tikv/issues/7295) and join us by picking the task you are interested in.
```
Comment (nrc) [inline content/blog/monthly-april-2020.md:13]:
```suggestion
We're working on another API - VerKV, a multi-versioned KV without transactions, to help TiKV expand application scenarios. Its an API halfway between RawKV and TxnKV. The work plan proposed by [@fredchenbj](https://github.com/fredchenbj) is available [here](https://github.com/tikv/tikv/issues/7295); join us by picking a task you are interested in!
```

Could also add a link to the RFC: https://github.com/tikv/rfcs/blob/master/text/2020-03-11-versioned-kv.md

File: content/blog/monthly-april-2020.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+It is halfway through the [TiDB Usability Challenge](https://pingcap.com/blog/tidb-usability-challenge-dare-to-dream-bigger/) and we are glad to see TiKV related projects are popular. Congratulations to the teams that grabbed scores and let’s take a look at the top 5 most scored projects in TiKV repo.
```
Comment (nrc) [inline content/blog/monthly-april-2020.md:15]:
```suggestion
It is halfway through the [TiDB Usability Challenge](https://pingcap.com/blog/tidb-usability-challenge-dare-to-dream-bigger/) and we are glad to see TiKV related projects are popular. Congratulations to the teams that grabbed scores! Let’s take a look at the top 5 most scored projects in TiKV repo:
```

File: content/blog/monthly-april-2020.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+## Releases
+
+This month our team made 5 TiKV releases!
+
+You can review the changelogs here:
+
+*   [3.1.0-rc](https://github.com/tikv/tikv/releases/tag/v3.1.0-rc)
+    *   New features
+        *   Support backing up data with the Raw KV API in [#7051](https://github.com/tikv/tikv/pull/7051)
+        *   Support the Transport Layer Security (TLS) authentication for the status server in [#7142](https://github.com/tikv/tikv/pull/7142)
```
Comment (nrc) [inline content/blog/monthly-april-2020.md:36]:
```suggestion
        *   Support TLS authentication for the status server in [#7142](https://github.com/tikv/tikv/pull/7142)
```

File: content/blog/monthly-april-2020.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        *   Support the TLS authentication for the KV server in [#7305](https://github.com/tikv/tikv/pull/7305)
```
Comment (nrc) [inline content/blog/monthly-april-2020.md:37]:
```suggestion
        *   Support TLS authentication for the KV server in [#7305](https://github.com/tikv/tikv/pull/7305)
```

File: content/blog/monthly-april-2020.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    *   Bug fixes
+*   [4.0.0-rc](https://github.com/tikv/tikv/releases/tag/v4.0.0-rc)
+    *   New features
+        *   Support the `pipelined` feature in pessimistic transactions in [#6984](https://github.com/tikv/tikv/pull/6984)
+        *   Support TLS in the HTTP port in [#5393](https://github.com/tikv/tikv/pull/5393)
+        *   Enable the `unify-read-pool` configuration item in new clusters by default in [#7059](https://github.com/tikv/tikv/pull/7059)
+    *   Bug fixes
+*   [3.1.0](https://github.com/tikv/tikv/releases/tag/v3.1.0)
```
Comment (nrc) [inline content/blog/monthly-april-2020.md:45]: We should make this one stand out a bit more since it is a new 'full' release

**Comment by WT-Liu** [comment]
@dcalvin PTAL again~ Thanks!

**Pull request merged**